addons/escoria-core/game/core-scripts/esc_item.gd

```diff
@@ -379,14 +379,21 @@ func get_animation_player() -> Node:
 #
 # **Returns** The interaction position
 func get_interact_position() -> Vector2:
+	var multiple_positions_found = false
 	var interact_position = null
 	for c in get_children():
 		if c is Position2D:
+			if interact_position != null:
+				multiple_positions_found = true
 			interact_position = c.global_position
 
 	if interact_position == null and collision != null:
 		interact_position = collision.global_position
 
+	if multiple_positions_found == true:
+		escoria.logger.warning("Multiple ESClocations found to walk to for object " +  global_id + \
```

Comment:
Prefer string interpolation to concatenation; i.e. make use of the `%` operator.

Reply:
It didn't work when I tried it as string interpolation.

Reply:
It didn't work when I tried it as string interpolation.

Reply:
Not sure why it wouldn't...what was the exact line? I know that you can for sure use string interpolation like:

```
var s = "here"
escoria.logger.warning("Using interpolation %s" % s)
```

Reply:
My mistake, had a comma instead of percent when I coded it the first time. Oops 🤦